examples/pinterest.amp.html

```diff
@@ -71,11 +71,11 @@ <h3>Embedded Pins - responsive</h3>
       </amp-pinterest>
     </div>
 
-  <h2>Pin It Buttons</h2>
+  <h2>Save Buttons</h2>
 
   <h4>Small Gray English with No Count</h4>
 
-    <amp-pinterest height=20 width=40
+    <amp-pinterest height=20 width=50
```

Comment:
If it needs to be this much higher to render correctly I think it might be a breaking change, since the already-published amp documents will have the old dimensions in the markup.

Reply:
Ok, I added some commits that comply with the existing dimensions. Let me know how things look now.

Reply:
Nice, a side by side comparison looks really good!

Before|After
--|--
![localhost_8000_examples_pinterest amp html](https://user-images.githubusercontent.com/2363700/52420068-5124d800-2abf-11e9-92e6-fc78b79cfcfe.png)|![localhost_8000_examples_pinterest amp html 1](https://user-images.githubusercontent.com/2363700/52420079-56822280-2abf-11e9-8566-8941d3bcfe93.png)

Reply:
Where did you get the gray value for the gray buttons?

Reply:
I took it from the colors of the Follow Button since there is not brand guidelines on a standard gray value. Optionally I could also grab from the bubble text color (#555)?

Reply:
Yeah we kind of have to choose something ourselves. The current gray seems a little dark to me. I think the bubble text color could look better but maybe @andrewwatterson has an opinion on what gray would look best here

Reply:
Nice, love the side-by-side screenshot. 

On the widget, it still says "Pin", is that change not part of this PR? (if plan is to change that in a different PR, let's make sure they both go out in the same Canary cut.)

Regarding Gray, follow button I see on is a bit different. It has ```box-shadow: inset 0 0 1px #888; background-color: #efefef;``` I also see Pinterest no longer supporting `data-pin-color` and it is always red. It don't mind us doing the same and breaking color variable all together. At very minimum we should change the default to be red (right now it is gray)

Reply:
I wasn't planning on modifying the PinWidget in this PR but in retrospect it will be nice for these changes to be done as a unit. I'll send the new change shortly.

Reply:
How about gray with background #efefef and color #555, to match the bubble style? It's softer/easier to look at, though a case could be made it is more muted than we'd want it to be.
![image](https://user-images.githubusercontent.com/10456171/52436643-cf46a600-2ae2-11e9-872c-d5eda16c03bb.png)